src/main/java/hudson/plugins/accurev/CheckForChanges.java

```diff
@@ -295,4 +192,42 @@ public static boolean checkStreamForChanges(AccurevServer server,
         }
         return isTransLatestThanBuild;
     }
+
+    private static boolean changesMatchFilter(TaskListener listener, Collection<String> serverPaths, List<String> filters) {
+        if (filters == null || filters.isEmpty()) {
+            // No filters, so always a match.
+            return true;
+        }
+
+        for (String filterPath : filters) {
+            // Paths are sanitized using java.nio.file.Paths (makes sure that the same path separators are used)
+            final Path fp = Paths.get(filterPath);
+            for (String serverPath : serverPaths) {
+                final Path sp = Paths.get(serverPath);
```

Comment:
This is why I'm creating a Path object. It automatically converts the paths so that the slashes will be the same.
However, a string replace will probably work fine too, so I'm thinking of removing the Path object and only using the search & replace.